src/utils.py

```diff
@@ -10,7 +10,7 @@
 import github.Repository
 
 IMAGE_SUFFIXES = [
-    ".jpg", ".jpeg", ".png", ".tif", ".tiff", ".webp", ".gif", ".mp4"
+    ".jpg", ".jpeg", ".png", ".tif", ".tiff", ".webp", ".gif", ".gifv", ".mp4", ".webm"
```

Comment:
This would close #16 

```suggestion
    ".jpg", ".jpeg", ".png", ".tif", ".tiff", ".webp", ".gif", ".gifv", ".mp4", ".webm", ".mov"
```

Reply:
.mov is not supported on all devices which is why it was excluded.

Reply:
Fair point, but it's still a video, so why block it… Couldn't people download a player to play it? :thinking:

Reply:
I'm unsure if thumbnails are available on unsupported devices which would probably discouraged user interaction.

Assuming that someone does chose to watch the video. Default behaviour downloads the file which means it needs to be removed afterwards. Downloading an application to playback one video type that may still not be supported; not sure if it'll be hardware or software related but enough people have stated there is no workaround.

I understand that it is an embeddable format however we should focus on maximum inclusion. This would only encourage people to use more open and supported formats.